Dockerfile

```diff
@@ -1 +1,7 @@
 FROM hone/mruby-cli
+
+RUN groupadd -r mruby -g 1000 && \
+    useradd -d /home/mruby/ -u 1000 -g mruby -r -s /sbin/nologin -c "Docker image mruby user" mruby && \
+    chown -R mruby:mruby /home/mruby/
```

Comment:
@zzak is it the issue you mentioned https://github.com/boot2docker/boot2docker/issues/581 ?

by the way, new version of docker comes with interpolation, very helpful for a non static UID https://github.com/docker/compose/issues/1532#issuecomment-140160603